### PR TITLE
[11.0][ADD] hr_attendance_warning

### PR DESCRIPTION
**Pull request by jarroyomorales**

### CODE REVIEW

**Comment by jarroyomorales** [comment]
He añadido un system parameter que te permite activar o desactivar el widget ya que si esto va a la OCA, puede que alguien no quiera el icono en la systray

**Comment by jarroyomorales** [comment]
@etobella The warnings widget should be fixed now.

**Comment by jarroyomorales** [comment]
Fix problem with public holidays

File: hr_attendance_warning/data/ir_config_parameter_data.xml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data noupdate="1">
+        <record id="show_attendance_warning" model="ir.config_parameter">
+            <field name="key">show.attendance.warning</field>
+            <field name="value">True</field>
+        </record>
+    </data>
+</odoo>
```
Comment (HviorForgeFlow) [inline hr_attendance_warning/data/ir_config_parameter_data.xml]: Nit: Add end blank line

File: hr_attendance_warning/wizards/hr_attendance_warning_solve.py [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+from odoo import api, fields, models
+
+
+class HrAttendanceWarningSolve(models.TransientModel):
+    _name = 'hr.attendance.warning.solve'
+
+    name = fields.Char()
+
+    @api.multi
+    def doit(self):
```
Comment (HviorForgeFlow) [inline hr_attendance_warning/wizards/hr_attendance_warning_solve.py]: Improve method name

File: hr_attendance_warning/static/src/less/systray.less [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+    .o_warning_navbar_dropdown {
+        width: 350px;
+        padding: 0;
+    }
+}
+
+.o_view_all_warnings {
+    .btn-link;
+    padding: 5px;
+}
```
Comment (HviorForgeFlow) [inline hr_attendance_warning/static/src/less/systray.less]: Nit: Add end blank line.

**Review by HviorForgeFlow — COMMENTED** [review]
Didn't notice until now that this is not an OCA repo, but here we go :)

**Comment by jarroyomorales** [comment]
@hveficent  Pf yes, i wasnt sure about publishing it to OCA yet, there is a lot of duplicated code for check in and for check out, but I think tomorrow i will do a PR on this and lets see what people thinks.

**Pull request closed without merging**